src/api/common/schemas/ledger-closed.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "ledgerClosed",
+  "description": "A ledgerClosed event message",
+  "type": "object",
+  "properties": {
+    "feeBase": {"type": "integer", "minimum": 0},
+    "feeReference": {"type": "integer", "minimum": 0},
+    "ledgerHash": {"$ref": "ledgerHash"},
+    "ledgerVersion": {"$ref": "ledgerVersion"},
+    "ledgerTimestamp": {"type": "integer", "minimum": 0},
```

Comment:
I think so, otherwise we should think about exposing a convenience method (maybe via ripple-lib-extensions) for working with [Ripple Epoch time](https://ripple.com/build/rippled-apis/#specifying-time)